squad/api/rest.py

```diff
@@ -602,10 +602,25 @@ class ProjectStatusSerializer(serializers.HyperlinkedModelSerializer):
 
     def to_representation(self, instance):
         ret = super().to_representation(instance)
+        enriched_details = self.context.get('enriched_details', None)
         if instance.regressions is not None:
-            ret['regressions'] = json.dumps(instance.get_regressions())
+            regressions = instance.get_regressions()
+            if enriched_details:
+                for env in enriched_details.keys():
+                    env_regressions = regressions.get(env, None)
+                    if env_regressions:
+                        enriched_details[env].update({'regressions': env_regressions})
+            ret['regressions'] = json.dumps(regressions)
```

Comment:
Any reason why having regressions and fixes as json? The output will be a string with json in it

Reply:
@chaws good question...I should have reviewed this line and changed it ..it was already like this